### PR TITLE
#2: Solving Add Two Numbers Problem

### PR DESCRIPTION
**Pull request by DShydlouski**

### CODE REVIEW

File: ScnSoft-Education/LeetCodeTraining.Tests/AddTwoNumsTests.cs [hunk 1]
```diff
@@ -0,0 +1,62 @@
+using LeetCodeTraining.AddTwoNumbers;
+
+namespace LeetCodeTraining.Tests
+{
+    public class AddTwoNumsTests
+    {
+
```
Comment (dsubotkovsky) [inline ScnSoft-Education/LeetCodeTraining.Tests/AddTwoNumsTests.cs:7]: remove empty line

File: ScnSoft-Education/LeetCodeTraining/AddTwoNumbers/AddTwoNumbersSolution.cs [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+            this.val = val;
+            this.next = next;
+        }
+    }
+
+    public static class AddTwoNumbersSolution
+    {
+        public static ListNode AddTwoNumbers(ListNode l1, ListNode l2, int n = 0)
+        {
+            if (l1 == null && l2 == null && n == 0) return null;
```
Comment (dsubotkovsky) [inline ScnSoft-Education/LeetCodeTraining/AddTwoNumbers/AddTwoNumbersSolution.cs:18]: add {..} after if condition.
Use the same style code in all your tasks

File: ScnSoft-Education/LeetCodeTraining/AddTwoNumbers/AddTwoNumbersSolution.cs [hunk 3]
```diff
@@ -0,0 +1,26 @@
+﻿namespace LeetCodeTraining.AddTwoNumbers
+{
+    public class ListNode
+    {
+        public int val;
+        public ListNode next;
```
Comment (dsubotkovsky) [inline ScnSoft-Education/LeetCodeTraining/AddTwoNumbers/AddTwoNumbersSolution.cs:6]: add an empty line after fields declaration

File: ScnSoft-Education/LeetCodeTraining.Tests/AddTwoNumsTests.cs (hunk 1 above)
Comment (DShydlouski) [reply ScnSoft-Education/LeetCodeTraining.Tests/AddTwoNumsTests.cs]: Empty line was removed

File: ScnSoft-Education/LeetCodeTraining/AddTwoNumbers/AddTwoNumbersSolution.cs (hunk 2 above)
Comment (DShydlouski) [reply ScnSoft-Education/LeetCodeTraining/AddTwoNumbers/AddTwoNumbersSolution.cs]: Brackets were added

File: ScnSoft-Education/LeetCodeTraining/AddTwoNumbers/AddTwoNumbersSolution.cs (hunk 3 above)
Comment (DShydlouski) [reply ScnSoft-Education/LeetCodeTraining/AddTwoNumbers/AddTwoNumbersSolution.cs]: Empty line was added

File: ScnSoft-Education/LeetCodeTraining/AddTwoNumbers/AddTwoNumbersSolution.cs [hunk 4]
```diff
@@ -0,0 +1,26 @@
+﻿namespace LeetCodeTraining.AddTwoNumbers
+{
+    public class ListNode
```
[the remark this answers is not in the document]
Comment (DShydlouski) [reply ScnSoft-Education/LeetCodeTraining/AddTwoNumbers/AddTwoNumbersSolution.cs]: Class were moved to separate file

**Pull request merged**